### PR TITLE
Monthly

### PR DESCRIPTION
**Pull request by LFISHER7**
This adds in functionality to change the time window for aggregated counts to be 4-weekly if the time period is > 52 weeks.

### CODE REVIEW

File: analysis/study_utils.py [hunk 1]
```diff
@@ -213,3 +213,25 @@ def redact_events_table(events_counts, low_count_threshold, rounding_base):
     )
 
     return events_counts
+
+
+def convert_weekly_to_monthly(counts_table):
```
Comment (inglesp) [inline analysis/study_utils.py:218]: Could you add a docstring to say that we're grouping into periods of four weeks, instead of one month?

**Review by inglesp — APPROVED** [review]
Looks good!

Have we considered whether we need to scale up the count in the last month, to account for it being shorter than the previous months?

**Comment by LFISHER7** [comment]
This now removes weeks from the beginning of the time period if the number of weeks is not a multiple of 4 as an alternative to scaling the last month

**Review by lucyb — COMMENTED** [review]
Looks good.

**Pull request merged**